packages/side-nav/src/CollaspeToggle/CollapseToggle.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import ChevronLeft from '@leafygreen-ui/icon/dist/ChevronLeft';
+import ChevronRight from '@leafygreen-ui/icon/dist/ChevronRight';
+import Tooltip from '@leafygreen-ui/tooltip';
+import { InlineKeyCode } from '@leafygreen-ui/typography';
+
+import { useSideNavContext } from '../SideNav/SideNavContext';
+
+import {
+  buttonBaseStyles,
+  buttonCollapsedStyles,
+  buttonThemeStyles,
+  iconClassName,
+  iconWrapper,
+  keyboardShortcut,
+} from './styles';
+import { CollapseToggleProps } from './types';
+
+/**
+ * @internal
+ */
+export function CollapseToggle({
+  className,
+  collapsed,
+  hideTooltip,
+  ...rest
+}: CollapseToggleProps) {
+  const { navId, darkMode, theme } = useSideNavContext();
+
+  const Chevron = collapsed ? ChevronRight : ChevronLeft;
+
+  return (
+    <Tooltip
+      darkMode={darkMode}
+      align="right"
+      justify="middle"
+      open={typeof hideTooltip === 'boolean' ? !hideTooltip : undefined}
+      trigger={
+        <button
+          data-testid="side-nav-collapse-toggle"
+          aria-label="Collapse navigation"
+          aria-controls={navId}
+          aria-expanded={!collapsed}
+          className={cx(
+            buttonBaseStyles,
+            buttonThemeStyles[theme],
+            {
+              [buttonCollapsedStyles]: collapsed,
+            },
+            className,
+          )}
+          {...rest}
+        >
+          <div className={cx(iconClassName, iconWrapper)}>
+            <Chevron role="presentation" />
+          </div>
+        </button>
+      }
+    >
+      <span aria-hidden>
+        {collapsed ? 'Expand' : 'Collapse'}
+        <InlineKeyCode className={keyboardShortcut} darkMode={!darkMode}>
```

Comment:
Do we have a ticket to fix InlineKeyCode so that it can consume darkMode the same way our other components do?

Reply:
Oh this is an error on my end, I shouldn't have to specify darkMode. I'll remove.